keps/sig-network/2433-topology-aware-hints/README.md

```diff
@@ -304,17 +289,56 @@ and disabled states. Without this fallback, endpoints could easily get
 overloaded as hints were being added or removed from some EndpointSlices but
 had not yet propagated to all of them.
 
+Note: Some future heuristics may not rely on hints and could instead be
+implemented directly by kube-proxy.
+
 ### EndpointSlice Controller
 
 When the `TopologyAwareHints` feature gate is enabled and the annotation is set
-to `Auto` for a Service, the EndpointSlice controller will add hints to
-EndpointSlices. These hints will indicate where an endpoint should be consumed
-by proxy implementations to enable topology aware routing.
+to `Auto` or `ProportionalCPU` for a Service, the EndpointSlice controller will add
+hints to EndpointSlices. These hints will indicate where an endpoint should be
+consumed by proxy implementations to enable topology aware routing.
+
+## Heuristics
+
+This KEP starts with the following heuristics:
+
+* Auto: EndpointSlice controller and/or underlying dataplane can choose the
+        heuristic used.
+* ProportionalCPU: Endpoints will be allocated to each zone proportionally,
```

Comment:
ProportionalByCore ?

CoreProportionalSafe?

Should it capture the idea of proprotional, safety, and zone, maybe?

ZoneProportionalSafe ?

ZoneCoreProportionalSafe ?

Reply:
I'd hate to add "Safe" to anything that actually could increase risk compared to the status quo. I like the idea of `ProportionalByCore`. If we end up with more aggressive/risky heuristics, I think it would make sense to add something like an `Aggressive` or `Strict` suffix to those.

Reply:
Updated to ProportionalByCore.